apis/v1/objectstorage_types.go

```diff
@@ -19,7 +19,7 @@ type BucketDeletionPolicy string
 
 //go:generate yq -i e ../generated/appcat.vshn.io_objectbuckets.yaml --expression "with(.spec.versions[]; .schema.openAPIV3Schema.properties.spec.properties.parameters.properties.security.default={})"
 // +kubebuilder:object:root=true
-// +kubebuilder:printcolumn:name="Bucket Name",type="string",JSONPath=".spec.parameters.bucketName"
+// +kubebuilder:printcolumn:name="Bucket Name",type="string",JSONPath=".status.bucketName"
```

Comment:
Does that even work? Afaik when kubebuilder runs it doesn't have any information on the `.status` fields